build/repo.props

```diff
@@ -15,9 +15,9 @@
 
     <SubmoduleRoot>$(RepositoryRoot)src\</SubmoduleRoot>
 
-    <ProductPackageOutputPath>$(ArtifactsDir)$(Configuration)\packages\product\</ProductPackageOutputPath>
-    <InternalPackageOutputPath>$(ArtifactsDir)$(Configuration)\packages\internal\</InternalPackageOutputPath>
-    <InstallersOutputPath>$(ArtifactsDir)$(Configuration)\installers\</InstallersOutputPath>
+    <ArtifactsShippingPackagesDir>$(ArtifactsDir)\packages\$(Configuration)\Shipping\</ArtifactsShippingPackagesDir>
```

Comment:
Suggest moving these properties to one file and importing that from here, the root `Directory.Build.props` file and anywhere else they're needed e.g. the templating `Directory.Build.props` file

Reply:
What's the connection? (These properties aren't framework-specific or conditional, `%(ArtifactsDir)` could be moved to the common `.props` file, and `%(Configuration)` is fine.)

Reply:
Once we do #4247, we will be able to build the shared framework without having to build, pack, and then restore against the packages we just made.

> %(ArtifactsDir) could be moved to the common 

It could be, but I'm trying to reduce the number of things imported into the KoreBuild context and push as much down into the project-context as possible.

By the way, variables use the $ sign - `$(ArtifactsDir)`. Percent sign `%(` means batch an item group.

Reply:
Got it. And, I apologize for my fumble with the `%`-sign. I'll edit my comment 😈